ballerina/sql_client.bal

```diff
@@ -131,37 +176,41 @@ public client class SQLClient {
         _ = check self.dbClient->execute(query);
     }
 
-    public isolated function checkExists(map<anydata>? filter) returns error? {
-        sql:ParameterizedQuery query = sql:queryConcat(`SELECT * FROM `, self.tableName, stringToParameterizedQuery(" " + self.entityName));
-
-        if !(filter is ()) {
-            query = sql:queryConcat(query, ` WHERE`, check self.getWhereClauses(filter));
-        }
-
-        _ = check self.dbClient->execute(query);
-    }
-
-    public isolated function getManyRelations(record {} result, string[] include) returns error? {
+    # Retrieves the values of the 'many' side of an association.
+    #
+    # + 'object - The record to which the retrieved records should be appended
+    # + include - The relations to be retrieved (SQL `JOINs` to be performed)
+    # + return - `()` if the operation is performed successfully or an `error` if the operations fails.
```

Comment:
```suggestion
    # + return - `()` if the operation is performed successfully or an `error` if the operations fail
```

Reply:
Fixed in [83ba8ac](https://github.com/ballerina-platform/module-ballerina-persist/pull/29/commits/83ba8aca943aaaaffd88f25e28fe62a5c00c1ef6)